python/py-doxyqml/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               gitlab 1.0
+
+gitlab.instance         https://invent.kde.org
+gitlab.setup            sdk doxyqml 0.5.3 v
+
+name                    py-doxyqml
+categories-append       textproc
+platforms               {darwin any}
+supported_archs         noarch
+license                 BSD
+maintainers             nomaintainer
+
+description             A filler for Doxygen to document QML files
+long_description        ${name} is an input filler for Doxygen, a documentation \
+                        system for C++ and a few other languages. ${name} makes \
+                        it possible to use Doxygen to document QML code.
+
+checksums               rmd160  dbb9cd9b5d75c18cb9358703b9097c5e6c06ba8b \
+                        sha256  4b544945c76fcdadf8b17b7cccb1ccf59edd01ffb9bd0190602308d21025d139 \
+                        size    23859
+
+python.versions         38 39 310 311 312
+python.default_version  311
```

Comment:
please don't set this explicitly, it's done in the python PG.

Reply:
I didn't check, but is this intended to be used as a package by other Python packages (i.e., imported as a module)? Or is it just a program that happens to be written in Python and the actual Python version used doesn't matter?

In the latter case, the name should not start with "py-" and instead of "python.versions" it should just set the "python.default_version". 

Sorry, that should have been my initial review comment 😉.